plugins_/command_completions/builtin_commands_meta_data.yaml

```diff
@@ -69,16 +142,32 @@ delete_word:
   args:
     forward: true
     sub_words: false
+detect_indentation:
+  args:
+    show_message: false
+  command_type: text
+  doc_string: Guess indentation settings from the text content.
+  added: 4050
```

Comment:
Hm, the command existed before but as a Python implementation. We don't interpret the `added` data embedded into this document yet, but maybe we should to prevent duplicates.

Reply:
There are also some functions which have been removed, like `insert_best_completions`. There are only few of them, but maybe should mark those as `removed` in the same way?

Reply:
Imo, yes. My hope is that we can use this one location for the command metadata both for PD and some documentation, e.g. the unofficial ones. Not currently, of course, but it makes sense to add such metadata when you notice it or know about it instead of trying to fill back the gaps later.